.github/workflows/mobilecoin-dev-cd.yaml

```diff
@@ -0,0 +1,563 @@
+        # rust-sgx-base changes RUSTUP_HOME and CARGO_HOME to workaround GHA overriding HOME
+        # Key is a hash of all the Cargo.lock files. If lock files change, invalidate cache so build pulls fresh packages.
+        path: |
+          /opt/cargo/git
+          /opt/cargo/registry/index
+          /opt/cargo/registry/cache
+        key: ${{ secrets.CACHE_BUSTER }}-${{ runner.os }}-${{ hashFiles('**/Cargo.lock') }}-cargo-cache
+
+    - name: Write environment values
+      env:
+        ENCLAVE_SIGNING_KEY_PATH: ${{ github.workspace }}/.tmp/enclave_signing.pem
+        ENCLAVE_SIGNING_KEY: ${{ secrets.ENCLAVE_SIGNING_KEY }}
+      run: |
+        mkdir -p ${GITHUB_WORKSPACE}/.tmp
+        echo "${ENCLAVE_SIGNING_KEY}" > "${ENCLAVE_SIGNING_KEY_PATH}"
+
+    - name: Build Rust
+      if: |
+        steps.artifact_cache.outputs.cache-hit != 'true' &&
+        ! contains(github.event.head_commit.message, '[skip build]')
+      env:
+        IAS_MODE: DEV
+        SGX_MODE: HW
+        RUST_BACKTRACE: full
+        MOB_RELEASE: 1
+        CONSENSUS_ENCLAVE_PRIVKEY: ${{ github.workspace }}/.tmp/enclave_signing.pem
+        LEDGER_ENCLAVE_PRIVKEY: ${{ github.workspace }}/.tmp/enclave_signing.pem
+        VIEW_ENCLAVE_PRIVKEY: ${{ github.workspace }}/.tmp/enclave_signing.pem
+        INGEST_ENCLAVE_PRIVKEY: ${{ github.workspace }}/.tmp/enclave_signing.pem
+      run: |
+        cargo build --release \
+          -p mc-consensus-service \
+          -p mc-admin-http-gateway \
+          -p mc-util-generate-sample-ledger \
+          -p mc-util-grpc-admin-tool \
+          -p mc-util-keyfile \
+          -p mc-ledger-distribution \
+          -p mc-ledger-migration \
+          -p mc-ledger-from-archive \
+          -p mc-mobilecoind \
+          -p mc-watcher \
+          -p mc-fog-ingest-server \
+          -p mc-fog-ingest-client \
+          -p mc-fog-view-server \
+          -p mc-fog-report-server \
+          -p mc-fog-sql-recovery-db \
+          -p mc-fog-ledger-server \
+          -p mc-fog-test-client \
+          -p mc-fog-distribution \
+          -p mc-util-grpc-token-generator
+
+    - name: Copy artifacts to cache
+      if: |
+        steps.artifact_cache.outputs.cache-hit != 'true' &&
+        ! contains(github.event.head_commit.message, '[skip build]')
+      run: |
+        mkdir -p build_artifacts
+        find target/release -maxdepth 1 -executable -type f -exec cp "{}" build_artifacts/ \;
+        find target/release -maxdepth 1 -name "*.signed.so" -exec cp "{}" build_artifacts/ \;
+
+    - name: Create css measurements
+      if: |
+        steps.artifact_cache.outputs.cache-hit != 'true' &&
+        ! contains(github.event.head_commit.message, '[skip build]')
+      shell: bash
+      run: |
+        cd build_artifacts
+        for i in *.signed.so
+        do
+          css=$(echo -n "${i}" | sed -r 's/(.*)\.signed\.so/\1/')
+          sgx_sign dump -enclave "${i}" -dumpfile /dev/null -cssfile ${css}.css
+        done
```

Comment:
Yes, since the tools are installed we might as well create the .css files.  Just so the measurement files are readily available as an artifact for use in clients that might need them.